charts/kube-prometheus-stack/values.yaml

```
@@ -1464,6 +1464,11 @@ kubelet:
     additionalLabels: {}
     #  foo: bar
 
+    ## defines the labels which are transferred from the associated Kubernetes Pod object onto the ingested metrics.
```

```suggestion
    ## defines the labels which are transferred from the associated Kubernetes Service object onto the ingested metrics.
```

Sorry for the typo :D